### PR TITLE
Modified the add track to beacon UI

### PR DESCRIPTION
**Pull request by IB-12**
Reorganized the buttons and made the UI adapt to the viewmodel state

### CODE REVIEW

**Review by mbouaissi — CHANGES_REQUESTED** [review]
thx for this, can you also make so that there is only one plus button (if possible ofc)

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/BeaconScreen.kt [hunk 1]
```diff
@@ -131,18 +135,33 @@ fun BeaconInformation(location: Location) {
...
+            onClick = {
+              viewModel.changeChosenSongs()
+              chosenList.value =
+                  if (viewModel.isTopSongsListVisible.value) " Top Songs " else "Liked Songs"
+            }) {
+              Text(text = chosenList.value)
+            }
+        IconButton(
+            onClick = {
+              Log.d("AddTrack", "Adding track to beacon ${viewModel.beaconId}")
```
Comment (KlaKalma) [inline app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/BeaconScreen.kt:157]: Maybe delete these, no ?

**Review by KlaKalma — CHANGES_REQUESTED** [review]
Nice job, I'll try to do the back end as soon as possible !

File: app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/BeaconScreen.kt (hunk 1 above)
Comment (IB-12) [reply app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/BeaconScreen.kt:157]: Will do

**Comment by IB-12** [comment]
@KlaKalma Wdym by backend ? I already did it in #337

**Comment by KlaKalma** [comment]
> @KlaKalma Wdym by backend ? I already did it in #337

Yep sorry for the mistake I thought this was the add and ban track from the profile. Sorry for that.

**Review by KlaKalma — APPROVED** [review]
Good sir !

**Comment by IB-12** [comment]
Thanks !

**Pull request merged**